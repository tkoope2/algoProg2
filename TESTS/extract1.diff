9c9
< @     PATH: <>
---
> @     PATH: <c1 c2 c3 c4 b4 b1 b2 b3 a3 a4 a1>
11c11
< @     PATH: <>
---
> @     PATH: <c1 c2 c3 c4 b4 b1 b2 b3 a3 a4 a1 a2>
13c13
< @     PATH: <>
---
> @     PATH: <c1 c2 c3 c4 b4 b1 b2 b3 a3>
15c15
< @     PATH: <>
---
> @     PATH: <c1 c2 c3 c4 b4 b1 b2 b3 a3 a4>
17c17
< @     PATH: <>
---
> @     PATH: <c1 c2 c3 c4 b4 b1>
19c19
< @     PATH: <>
---
> @     PATH: <c1 c2 c3 c4 b4 b1 b2>
21c21
< @     PATH: <>
---
> @     PATH: <c1 c2 c3 c4 b4 b1 b2 b3>
23c23
< @     PATH: <>
---
> @     PATH: <c1 c2 c3 c4 b4>
25c25
< @     PATH: <>
---
> @     PATH: <c1>
27c27
< @     PATH: <>
---
> @     PATH: <c1 c2>
29c29
< @     PATH: <>
---
> @     PATH: <c1 c2 c3>
31c31
< @     PATH: <>
---
> @     PATH: <c1 c2 c3 c4>
