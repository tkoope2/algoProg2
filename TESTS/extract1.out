@1++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
@2
@3 COMMAND-LINE:
@4
@5    ./dfs rings.g c1 
@6
@7++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
@ a1
@     PATH: <>
@ a2
@     PATH: <>
@ a3
@     PATH: <>
@ a4
@     PATH: <>
@ b1
@     PATH: <>
@ b2
@     PATH: <>
@ b3
@     PATH: <>
@ b4
@     PATH: <>
@ c1
@     PATH: <>
@ c2
@     PATH: <>
@ c3
@     PATH: <>
@ c4
@     PATH: <>
